src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceSeparatorErrors.java

```diff
@@ -1,9 +1,9 @@
-package  com.puppycrawl.   tools.checkstyle.checks.whitespace;
+package  com.puppycrawl.   tools.checkstyle.checks.whitespace.singlespaceseparator;
 
 import java.util.List;
 import  java.util.Vector;
 
-public class      InputSingleSpaceErrors  {
+public class      InputSingleSpaceSeparatorErrors  {
```

Comment:
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:testCompile (default-testCompile) on project checkstyle: Compilation failure
[ERROR] /D:/checkstyleWorkspaceEclipse/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/singlespaceseparator/InputSingleSpaceSeparatorErrors.java:[6,8] class InputSingleSpaceErrors is public, should be declared in a file named InputSingleSpaceErrors.java

You are correct. I didn't think the compiler would require this and make it an error.
I am sorry for the trouble.

@wate123 Please keep this file and it's class as its original name.
@romani Just so we can get this done, we can change class, file, and violation in new PR when the main issue is closer to being done.